DirectProgramming/C++SYCL_FPGA/Tutorials/Features/experimental/streaming_data_interfaces/README.md

```diff
@@ -0,0 +1,272 @@
+
+```c++
+#include <sycl/ext/intel/prototype/pipes_ext.hpp>
+
+...
+
+using StreamingBeatDataT = sycl::ext::intel::experimental::StreamingBeat<unsigned char, true, true>;
+
+...
+
+StreamingBeatT out_beat(data, sop, eop, empty);
+SecondPipeInstance::write(out_beat);
+```
+
+### Read and Write APIs
+
+The read and write APIs are the same as for all pipes. See the IP Authoring Interfaces Overview Tutorial for more information.
+
+## Build the `Streaming Data Interfaces` Tutorial
+
+>**Note**: When working with the command-line interface (CLI), you should configure the oneAPI toolkits using environment variables. Set up your CLI environment by sourcing the `setvars` script in the root of your oneAPI installation every time you open a new terminal window. This practice ensures that your compiler, libraries, and tools are ready for development.
+>
+> Linux*:
+> - For system wide installations: `. /opt/intel/oneapi/setvars.sh`
+> - For private installations: ` . ~/intel/oneapi/setvars.sh`
+> - For non-POSIX shells, like csh, use the following command: `bash -c 'source <install-dir>/setvars.sh ; exec csh'`
+>
+> Windows*:
+> - `C:\Program Files(x86)\Intel\oneAPI\setvars.bat`
+> - Windows PowerShell*, use the following command: `cmd.exe "/K" '"C:\Program Files (x86)\Intel\oneAPI\setvars.bat" && powershell'`
+>
+> For more information on configuring environment variables, see [Use the setvars Script with Linux* or macOS*](https://www.intel.com/content/www/us/en/develop/documentation/oneapi-programming-guide/top/oneapi-development-environment-setup/use-the-setvars-script-with-linux-or-macos.html) or [Use the setvars Script with Windows*](https://www.intel.com/content/www/us/en/develop/documentation/oneapi-programming-guide/top/oneapi-development-environment-setup/use-the-setvars-script-with-windows.html).
+
+### On Linux*
+
+1. Change to the sample directory.
+2. Build the program for Intel® Agilex® 7 device family, which is the default.
+   ```
+   mkdir build
+   cd build
+   cmake ..
+   ```
+   > **Note**: You can change the default target by using the command:
+   >  ```
+   >  cmake .. -DFPGA_DEVICE=<FPGA device family or FPGA part number>
+   >  ```
+   >
+   > This tutorial is only intended for use in the IP Authoring flow and does not support targeting an explicit FPGA board variant and BSP.
+
+3. Compile the design. (The provided targets match the recommended development flow.)
+
+   1. Compile and run for emulation (fast compile time, targets emulated FPGA device).
+      ```
+      make fpga_emu
+      ```
+   2. Generate the optimization report.
+      ```
+      make report
+      ```
+   3. Compile and run for simulation (fast compile time, targets simulated FPGA device).
+      ```
+      make fpga_sim
+      ```
+   4. Compile for FPGA hardware (longer compile time, targets an FPGA device).
+      ```
+      make fpga
```

Comment:
The corresponding cmake target is missing

Reply:
fixed